db/seeds.rb

```diff
@@ -4,8 +4,7 @@
 user = User.find_or_initialize_by(:email => ENV['SEED_EMAIL'] || "admin@example.com")
 
 if user.persisted?
-  puts "User with email '#{user.email}' already exists, not seeding."
-  exit
+  raise "User with email '#{user.email}' already exists, not seeding."
```

Comment:
I don't understand this change, we always seed on starup of the docker containers, I think raising an exception will confuse the user that something when wrong.